articles/native-platforms/ios-objc/07-authorization.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Authorization
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to perform certain actions in the app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objc',
+  pkgFilePath: 'iOS/basic-sample-objc/ObjcSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+Many identity providers will supply access claims, like roles or groups, with the user. You can request these in your token by setting `scope: openid roles` or `scope: openid groups`. However, not every identity provider provides this type of information. Fortunately, Auth0 has an alternative to it, which is creating a rule for assigning different roles to different users.
+
+> This tutorial assumes that you've already read the [rules tutorial](06-rules.md) and you know how to implement a basic rule in your app.
+
+### Before starting
+
+It's required that you've got [Lock](https://github.com/auth0/Lock.iOS-OSX) integrated in your project. You can check out the [login tutorial](01-login.md) for more information about it.
+
+### 1. Create a Rule to assign roles
+
+First, you will create a rule that assigns your users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](https://manage.auth0.com/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
```

Comment:
We do not hard-code values like the dashboard URL. Instead we use variables where applicable, in this case you have to use the variable `uiURL` that holds the value `https://manage.auth0.com`.

So instead of `[new rule page](https://manage.auth0.com/#/rules/new)`
you should use `[new rule page](${uiURL}/#/rules/new)`

For more information: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables